src/components/externalLinks/externalLinks.scss

```diff
@@ -0,0 +1,223 @@
+                display: flex;
+                align-items: center;
+                width: 250px;
+                height: 32px;
+                position: relative;
+                margin-right: 12px;
+
+                .search__input {
+                    height: 32px;
+                    outline: none;
+                    border: solid 1px var(--N200);
+                    background-color: var(--N50);
+
+                    &:hover,
+                    &:focus {
+                        border: solid 1px var(--N400);
+                    }
+                }
+            }
+
+            .filters-wrapper {
+                .badge {
+                    margin-left: 4px;
+                }
+
+                .apply-filter {
+                    width: 100%;
+                    height: 28px;
+                }
+            }
+        }
+
+        .saved-filters__wrap {
+            padding: 16px 20px 0 0;
+            display: flex;
+            flex-wrap: wrap;
+
+            .saved-filter {
+                display: flex;
+                align-items: center;
+                border: solid 1px var(--N200);
+                background-color: #f7fafc;
+                text-transform: none;
+                padding-left: 6px;
+                padding-right: 6px;
+                margin: 0 12px 6px 0;
+                min-height: 24px;
+
+                .saved-filter-divider {
+                    height: 24px;
+                    width: 1px;
+                    background-color: var(--N200);
+                }
+
+                .saved-filter__clear-btn {
+                    border: none;
+                    margin: 0;
+                    margin-left: 4px;
+                    padding: 0;
+                    line-height: 0;
+                    background-color: #f7fafc;
+
+                    svg {
+                        stroke: var(--N600);
+
+                        path {
+                            fill: var(--N600);
+                        }
+                    }
+                }
+            }
+
+            .saved-filters__clear-btn {
+                background: none;
+                margin: 0 0 6px;
+                padding: 0;
+            }
+        }
+    }
+
+    .external-links {
+        width: 962px;
+        height: 697px;
+        background: #ffffff;
+        border: 1px solid #d0d4d9;
+        box-sizing: border-box;
+        border-radius: 8px;
+        margin-top: 16px;
+
+        .external-link {
+            padding: 12px 20px;
+
+            .external-link-actions {
+                display: none;
+            }
+
+            &:hover {
+                background-color: var(--N50);
+
+                .external-link-actions {
+                    display: flex;
+                }
+            }
+        }
+    }
+
+    .external-link__divider {
+        height: 1px;
+        width: 100%;
+        background-color: #EDF1F5;
+    }
+
+    .external-links__header {
+        height: 40px;
```

Comment:
can we minimize CSS here?

Reply:
Yes. Tried to keep the CSS to a minimum.